### DISCUSSION TITLE
Docker部署失败，安装页面打不开，求助

### DISCUSSION THREAD

**Discussion by zidanereal5** [langgenius/dify#3995] [General]
只修改了yml文件里的，nginx的映射端口，通过IP地址:6000/install访问，打不开页面
![image](https://github.com/langgenius/dify/assets/123045775/8502b031-bde9-477b-a0d6-2084bda7f99f)
![image](https://github.com/langgenius/dify/assets/123045775/b039edec-0931-46e7-a26f-2e84a50893ed)

**Comment by crazywoola** [comment]
Please make sure following our language policy #1542
